sessionctx/variable/session.go

```diff
@@ -690,6 +695,8 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		s.EnableRadixJoin = TiDBOptOn(val)
 	case TiDBEnableWindowFunction:
 		s.EnableWindowFunction = TiDBOptOn(val)
+	case TiDBOptJoinOrderAlgoThreshold:
+		s.TiDBOptJoinOrderAlgoThreshold = tidbOptPositiveInt32(val, DefTiDBOptJoinOrderAlgoThreshold)
```

Comment:
Should we also limit its upper bound?

Reply:
I think the `set it too large` can be treated as that we want every join order calculated by dp.
Though it not reasonable in a real environment, i think it's reasonable in logic...

Reply:
If it is larger than 64, will the DP algorithm panic?

Reply:
Changed.